README.md

```diff
@@ -245,9 +250,10 @@ interface XRCubeLayer extends XRLayer {
 // Graphics Bindings
 //
 
-dictionary XRWebGLLayerSourceInit {
+dictionary XRLayerInit {
```

Comment:
Shortened the name

Reply:
I'm 100% fine with this as long as we do a bit of validation to make sure it applies across all the layer types we want to support except projection, which feels like it's "special". Cube map is the only one I really question. We also want to make sure it applies to WebGPU as well (I'm pretty sure it does.)

Reply:
Should we make this inherit from `XRWebGLLayerInit`?

Or perhaps create a new superclass dictionary type that everyone can inherit from? Perhaps that can be called `XRLayerInit` instead?

Reply:
> Should we make this inherit from `XRWebGLLayerInit`?

We're missing a couple of field from `XRWebGLLayerInit`: `antialias` and `framebufferScaleFactor`

> perhaps create a new superclass dictionary type that everyone can inherit from? Perhaps that can be called XRLayerInit instead?

Do you want `XRWebGLLayerInit` inherit from this base class as well?

Reply:
Yeah, basically. We can make this change in the core spec and inherit from it here IMO. Doesn't have to be done now.

But it does mean that perhaps XRLayerInit should be reserved for the superclass.